projects/packages/forms/tests/php/contact-form/Contact_Form_Test.php

```diff
@@ -2468,7 +2468,7 @@ public function test_get_instance_from_jwt_returns_with_all_attribute_data() {
 				'hiddenField2' => 'value2',
 			), // Hidden fields to include in the form.
 			'stepTransition'         => 'fade-slide',
-			'connectMailPoet'        => false,
+			'mailpoet'               => '',
```

Comment:
Tests were failing because we've updated the name of the MailPoet attribute from connectMailPoet to mailpoet. I've updated that here.